cmd/client-fs.go

```diff
@@ -451,7 +451,7 @@ func (f *fsClient) Copy(source string, size int64, progress io.Reader) *probe.Er
 
 // get - get wrapper returning reader and additional metadata if any.
```

Comment:
The comment on additional metadata is superfluous with this change, please remove it.